client/src/Components/LoginPage/Login.tsx

```diff
@@ -0,0 +1,291 @@
+          marginRight : 15
+        }} >
+        <TouchableOpacity onPress={()=>navigation.navigate("ForgotPassword")} >
+                <CustomText 
+                    style={{
+                      color : theme.fontColor
+                    }}
+                    fontSize={16}
+                    fontWeight='500'
+                 >Forgot Password ?</CustomText>
+            </TouchableOpacity>
+        </CustomView>
+
+
+      <CustomInput 
+        style={{
+          borderColor : theme.fontColor,
+          marginBottom:30
+        }}
+        height={50}
+        margin={12}
+        marginRight={30}
+        borderRadius={10}
+        borderWidth={2}
+        padding={10}
+        placeholder="*********" 
+        placeholderTextColor={getThemeStore().isDarkThemeEnabled.get() ? "white" : "black"}
+        value={getLoginStore().user_password.get()}
+        onChangeText={getLoginStore().setPassword}
+        keyboardType="default"
+        secureTextEntry={true}
+        />
+       
+        <CustomView 
+          style={{
+            display: 'flex',
+            justifyContent: 'center',
+            alignItems: 'center',
+
+            marginRight:10
+          }}
+        >
+            <CustomButton onPress={handleLogin}
+                style={{
+                  backgroundColor: theme.borderColor,
+                  marginBottom: 5,
+                  borderRadius: 30
+                }}
+                height={60}
+                width={350}
+                
+            >
+                <CustomView 
+                    style={{display:'flex' , justifyContent:'center' , alignContent:'center'}}
+                >
+                    <CustomText
+                        style={{
+                          color: 'black',
+                          marginTop:20,
+                          marginLeft:140
+                        }}
+                        fontSize={20}
+                        fontWeight='300'
+                    >Login</CustomText>
+                </CustomView>
+            </CustomButton>
+          </CustomView>
+          <CustomView
+            style={{
+              display: 'flex',
+              justifyContent: 'center',
+              alignItems: 'center'
+            }}
+          >
+            <CustomText
+              style={{
+                color : theme.fontColor,
+                paddingTop:5,
+                paddingBottom:20,
+                marginRight:25
+              }}
+              fontSize={15}
+              fontWeight='600'
+            >
+            _______ Or sign in with _______
+            </CustomText>
+          </CustomView>
+
+
+          <CustomView 
+          style={{
+            display: 'flex',
+            justifyContent: 'center',
+            alignItems: 'center',
+            paddingBottom:0,
+            marginRight:10
+          }}
+        >
+            <CustomButton onPress={()=>{Alert.alert("Google Sign In not activated right now!")}}
```

Comment:
split this component into multiple smaller componntes

Reply:
DONE ✅